src/utilities/util_dump.c

```diff
@@ -154,8 +155,11 @@ util_dump(WT_SESSION *session, int argc, char *argv[])
          * nothing will be visible. The only exception is if we've supplied a timestamp in which
          * case, we're specifically interested in what is visible at a given read timestamp.
          */
-        if (WT_STREQ(simpleuri, WT_HS_URI) && timestamp == NULL)
-            F_SET(cursor, WT_CURSTD_IGNORE_TOMBSTONE);
+        if (WT_STREQ(simpleuri, WT_HS_URI) && timestamp == NULL) {
+            dump_cursor = (WT_CURSOR_DUMP *)cursor;
+            /* Set the "ignore tombstone" flag on the underlying cursor. */
+            F_SET(dump_cursor->child, WT_CURSTD_IGNORE_TOMBSTONE);
+        }
```

Comment:
Should we change this line as well?

Reply:
Hmm, I'm not sure why we bother doing that. I may as well though.

Reply:
f32d3d5